mobile/src/main/java/org/hawkular/client/android/fragment/AlertsFragment.java

```diff
@@ -397,6 +401,8 @@ private void tearDownState(Bundle state) {
     public void onDestroyView() {
         super.onDestroyView();
 
+        isAlertsFragmentAvailableForAlerts = false;
```

Comment:
@garg-anuj I kept the `ForAlerts` part in the TriggersFragment to take out any ambiguity as the fragment is not only used for Alerts but the variable is only used for Alerts and kept `ForAlerts` part in the AlertFragment to maintain consistency.